.changeset/calm-dolphins-joke.md

```diff
@@ -0,0 +1,5 @@
+---
+'@techdocs/cli': minor
+---
+
+Change from cypress-plugin-snapshots to cypress-plugin-visual-regression-diff
```

Comment:
Is cypress settings or configuration something that's shipped with this package? If not I don't think we need to include this changeset either.